src/main/java/org/amahi/anywhere/activity/OfflineFilesActivity.java

```diff
@@ -180,7 +182,23 @@ private View getParentView() {
 
     @Subscribe
     public void onFileDeleting(OfflineFileDeleteEvent event) {
-        deleteFileFromOfflineStorage(event.getFile());
+        AlertDialog.Builder builder = new AlertDialog.Builder(this);
+        builder.setTitle("Delete file");
```

Comment:
I would recommend putting these strings visible to the user in strings.xml

Reply:
Okay, thanks. I will do that.